### DISCUSSION TITLE
Lack of compatibility mode causing input latency

### DISCUSSION THREAD

**Discussion by WojMen** [ppy/osu#19417] [Q&A] [answered]
### Type

Performance

### Bug description

I had 2 laptops and both of them couldn't properly work on osu stable without compatibility mode. I'm not sure but in osu lazer i can see that exclusive fulscreen is not enabled. That cause input latency same in osu without compatibility mode. I tried osu lazer on both systems Windows 10/11 same issue.

So my problem is Osu Lazer probbably can't open in Exclusive Fullscreen without compatibility mode from osu stable.  

I have all drivers updated.

### Screenshots or videos

![image](https://user-images.githubusercontent.com/48603759/181259437-20c7a1ec-c247-4147-be16-db88abf1e620.png)

### Version

2022.723.0

### Logs

[performance.log](https://github.com/ppy/osu/files/9199359/performance.log)
[database.log](https://github.com/ppy/osu/files/9199375/database.log)
[network.log](https://github.com/ppy/osu/files/9199376/network.log)
[runtime.log](https://github.com/ppy/osu/files/9199377/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9199378/updater.log)

**Comment by peppy** [comment, marked as answer]
If you use search you'll find many issues like this. We are aware and already tracking this. Please use stable for now.
